src/components/Select/src/SelectControl.vue

```diff
@@ -159,7 +159,7 @@ export default {
 	--color-accent: #222;
 	--color-border: transparent;
 	--color-border-active: #222;
-	--color-error: #ff3b30;
+	--color-error: rgba(206, 50, 23, 1);
```

Comment:
Why do we need to copy these values over and over again? It would be good to have a centralized variables file for colors that are universal throughout the framework.

Reply:
open issue: [Add ability to share CSS between components](https://github.com/square/maker/issues/39)

> This issue is available for anyone to take.

if you have the bandwidth then go for it

Reply:
Gotcha, I do not unfortunately